api/v1alpha1/virtualmachine_types.go

```diff
@@ -392,6 +392,33 @@ type NetworkInterfaceStatus struct {
 	IpAddresses []string `json:"ipAddresses,omitempty"`
 }
 
+// NetworkInterfaceStatusEx defines the properties of network interface extended status attached to the VirtualMachineTemplate
+// as seen by OVF properties
+type NetworkInterfaceStatusEx struct {
+	// MAC address of the network adapter
+	MacAddress string `json:"macAddress,omitempty"`
+
+	// Gateway setting
+	Gateway    string `json:"gateway,omitempty"`
+
+	// SubnetMask setting
+	SubnetMask string `json:"subnetMask,omitempty"`
+
+	// IpAddress represents zero or one IP address assigned to the network interface in CIDR notation.
+	// eg, "192.0.2.1/16".
+	IpAddress         string `json:"ip,omitempty"`
```

Comment:
Out of curiosity, is this file `go-fmt`'ed? The indentation between the three fields looks a little off.